googlemaps/elevation.py

```diff
@@ -16,41 +16,37 @@
 #
 
 """Performs requests to the Google Maps Elevation API."""
+
 from googlemaps import convert
 
+
 def elevation(client, locations):
     """
     Provides elevation data for locations provided on the surface of the
     earth, including depth locations on the ocean floor (which return negative
     values)
 
-    :param locations: A single latitude/longitude tuple or dict, or a list of
-            latitude/longitude tuples or dicts from which you wish to calculate
-            elevation data.
-    :type locations: list or tuple
+    :param locations: Array of latitude/longitude values from which you wish
+        to calculate elevation data.
+    :type locations: a single location, or a list of locations, where a
+        location is a string, dict, list, or tuple
 
     :rtype: list of elevation data responses
     """
-    params = {}
-    if type(locations) is tuple:
-        locations = [locations]
-
-    params["locations"] = convert.join_list("|",
-            [convert.latlng(k) for k in convert.as_list(locations)])
-
+    params = {"locations": convert.location_list(locations)}
```

Comment:
Again like the other newline point, I think this is superfluous. Neither pep8 or our internal guide have a say on it, but there's a ton of major open source code and internal code that defines dicts in  one line if they're short enough.